db/migrate/20220824090500_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
```

Comment:
# Congratulations: Your Postgres database is well set up. You ran the necessary migration files
-
![Capture](https://user-images.githubusercontent.com/63972670/186399358-dc3f2acc-1c91-4792-a94c-a1c70b63b0c6.JPG)